zproject/urls.py

```diff
@@ -361,8 +361,10 @@
 
     url(r'^accounts/login/sso/$', zerver.views.auth.remote_user_sso, name='login-sso'),
     url(r'^accounts/login/jwt/$', zerver.views.auth.remote_user_jwt, name='login-jwt'),
-    url(r'^accounts/login/social/(\w+)$', zerver.views.auth.start_social_login,
+    url(r'^accounts/login/social/(?P<backend>\w+)$', zerver.views.auth.start_social_login,
```

Comment:
If we split changes like this tweak to the existing `URLs` out into its own commit preparatory, we can merge that hunk and move this patchset more towards something that just adds code (which will help make it lower-risk).

Reply:
I think this is purely esthetical and does not change how the code works. I'll probably change that back to the original version in the next iteration

Reply:
Cool, I was just thinking you could put this kind of cleanup in its own commit at the bottom of the branch to make things a bit simpler.